clients/src/test/java/org/apache/kafka/clients/producer/internals/SenderTest.java

```diff
@@ -778,22 +778,22 @@ public void testIdempotenceWithMultipleInflights() throws Exception {
         prepareAndReceiveInitProducerId(producerId, Errors.NONE);
         assertTrue(transactionManager.hasProducerId());
 
-        assertEquals(0, transactionManager.sequenceNumber(tp0).longValue());
+        assertEquals(0, transactionManager.sequenceNumber(tp0));
```

Comment:
why were these longs in the first place?

Reply:
I can't think of a good reason.